cpp/benchmarks/common/generate_input.cu

```diff
@@ -83,6 +83,8 @@ T get_distribution_mean(distribution_params<T> const& dist)
   }
 }
 
+size_t avg_element_size(data_profile const& profile, cudf::data_type dtype);
```

Comment:
Why is it defined in .cu file and implemented elsewhere? If you need to call it locally, just make an inline function.